### PR TITLE
Fix for the issues where error/validation responses are not showing up

### PR DESCRIPTION
**Pull request by tsatyadevreddy**
Fix for wrapping of error/validation messages that are happening.

### CODE REVIEW

File: lib/remotipart/render_overrides.rb [hunk 1]
```diff
@@ -13,8 +13,7 @@ def self.included(base)
     def render_with_remotipart *args
       render_without_remotipart *args
       if remotipart_submitted?
-        textarea_body = response.content_type == 'text/html' ? html_escape(response.body) : response.body
-        response.body = %{<textarea data-type=\"#{response.content_type}\" data-status=\"#{response.response_code}\" data-statusText=\"#{response.message}\">#{textarea_body}</textarea>}
+        response.body = %{<script type=\"text/javascript\">try{}catch(err){}</script>#{response.body}}
```
Comment (bsimpson) [inline lib/remotipart/render_overrides.rb]: I don't understand the try/catch block without anything inside of it? What would this catch?

Comment (bsimpson) [inline lib/remotipart/render_overrides.rb]: Should `response.body` be inside of it?

Comment (tsatyadevreddy) [inline lib/remotipart/render_overrides.rb]: @bsimpson We don't need the try..catch here i just left it as is from the remotipart master. If at all we need it we can use it.

**Comment by bsimpson** [comment]
Good to merge

**Pull request merged**